.github/workflows/ci.yaml

```
@@ -25,7 +25,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           requireScope: true
-          subjectPattern: ^([A-Z]).+$
+          subjectPattern: "^.+$"
```

This is needed to enforce the commit subject being Sentence case because of the `CHANGELOG.md` format in Knope, if you check you will see all the commits now on main has this format.
```suggestion
          subjectPattern: ^([A-Z]).+$
```